src/views/tutorial-library/constants.ts

```diff
@@ -0,0 +1,40 @@
+import { productSlugsToNames } from 'lib/products'
+
+/**
+ * The Algolia index we are searching against for tutorials
+ */
+export const INDEX_NAME = 'prod_LEARN'
+
+/**
+ * Duration used to throttle search requests to avoid excessive network calls
+ */
+export const SEARCH_TIMEOUT_MS = 200
+
+/**
+ * Controls the resources filters, attribute is the name on the indexed tutorial object in Algolia
+ */
+export const RESOURCES = [
+	{
+		label: 'Video',
+		attribute: 'hasVideo',
+	},
+	{ label: 'Interactive', attribute: 'isInteractive' },
+]
+
+/**
+ * Controls the editions filters, values map to those specified on the indexed tutorial object in Algolia
+ */
+export const EDITIONS = [
```

Comment:
I wonder if we should map this to the `EditionOptions` enum somehow so the types remain in sync.